### DISCUSSION TITLE
No connect and RDP connect buttons

### DISCUSSION THREAD

**Discussion by garfield50** [amidaware/tacticalrmm#1899] [Q&A]
Hi everybody,

from last windows version we have 3 or 4 new computer who when i install windows agent, i have no connect and rdp connect button. Chrome, edge and firefox have same probleme.

I check on all computers, mesh agent and tacticalrmm service are working perfect.

when i try recover connection, restart connection and recover agent the probleme stay the same. I try to remove agent and reinstall without solve problem.

Thanks for your help.

![Capture d'écran 2024-06-21 114258](https://github.com/amidaware/tacticalrmm/assets/238294/07900558-1454-4c34-a33d-97f09b70d381)

**Comment by silversword411** [comment]
That means mesh agent isn't connecting to mesh server.

Run mesh troubleshooting scripts

**Comment by bbrendon** [comment]
I think that could also mean you don't have permission in mesh to the computer.

**Comment by garfield50** [comment]
I m not find the mesh script. But i found my problem:

I clear the cache into tools menu

in globals settings, i uncheck "Sync Mesh Perms with TRMM" in meshcentral section

thanks for your help :)

**Reply by alencarwell** [reply]
i have same problem, but dont working for me, helpme ?
